File: Service/Implement/DanhMucAPIService.cs
namespace Service.Implement
{
    public class DanhMucAPIService : BaseService<DanhMucAPI, IDanhMucAPIRepository>
    , IDanhMucAPIService
    {
    private readonly IDanhMucAPIRepository _DanhMucAPIRepository;
    public DanhMucAPIService(IDanhMucAPIRepository DanhMucAPIRepository) : base(DanhMucAPIRepository)
    {
    _DanhMucAPIRepository = DanhMucAPIRepository;
    }
    }
    }

File: Map/src/app/shared/DanhMucAPI.service.ts
﻿import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucAPI } from 'src/app/shared/DanhMucAPI.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucAPIService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucAPI";
    }
}

